runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -4877,6 +4935,7 @@ void amWrapPut(struct taskArg_RMA_t* tsk_rma) {
 
 static
 void amHandleAMO(struct amRequest_AMO_t* amo) {
+  DBG_PRINTF(DBG_AM | DBG_AM_RECV, "amHandleAMO result %p", amo->result);
```

Comment:
I wouldn't personally do this here. If we need/want debug output to confirm that the result address is being dealt with properly on the target node of the AMO, we should rely on `amRequestFAMOResult()` to print that for us.